server/keyshare/myirmaserver/server.go

```diff
@@ -86,6 +86,8 @@ func (s *Server) Handler() http.Handler {
 		router.Use(server.SizeLimitMiddleware)
 		router.Use(server.TimeoutMiddleware(nil, server.WriteTimeout))
 
+		router.Use(server.RecoverMiddleware)
```

Comment:
You've put this everywhere as first middleware to be applied. Maybe good to do that here as well. Then, also a panic in the other middlewares is included.

Reply:
Very true, and consistency is key, done